webapp/channels/src/components/view_user_group_modal/view_user_group_header_sub_menu/view_user_group_header_sub_menu.tsx

```diff
@@ -90,7 +90,7 @@ const ViewUserGroupHeaderSubMenu = (props: Props) => {
                 stopPropagationOnToggle={false}
                 id={`detailsCustomWrapper-${group.id}`}
             >
-                <button className='action-wrapper btn-icon'>
```

Comment:
Apparently, this `action-wrapper` class is only used here and in `webapp/channels/src/components/user_groups_modal/user_groups_list/user_groups_list.tsx`. But it is defined also in two places (`webapp/channels/src/components/user_groups_modal/user_groups_modal.scss` and `webapp/channels/src/components/view_user_group_modal/view_user_group_modal.scss`), so hopefully they are two different things. So...

- Can you check if the definition can be removed from `webapp/channels/src/components/view_user_group_modal/view_user_group_modal.scss`?
- Can you check if anything has to be done with the button in `webapp/channels/src/components/user_groups_modal/user_groups_list/user_groups_list.tsx` and therefore, if anything has to be removed from `webapp/channels/src/components/user_groups_modal/user_groups_modal.scss`?

Reply:
Done.